lib/octokit/client/organizations.rb

```diff
@@ -595,6 +595,87 @@ def remove_organization_membership(org, options = {})
         user = options.delete(:user)
         user && boolean_from_response(:delete, "orgs/#{org}/memberships/#{user}", options)
       end
+
+      # Initiates the generation of a migration archive.
+      #
+      # Requires authenticated organization owner.
+      #
+      # @param org [String, Integer] Organization GitHub login or id.
+      # @param repositories [Array<String>] :repositories Repositories for the organization.
+      # @option options [Boolean, optional] :lock_repositories Indicates whether repositories should be locked during migration
+      # @return [Sawyer::Resource] Hash representing the new migration.
+      # @see https://developer.github.com/v3/orgs/teams/#create-team
+      # @example
+      #   @client.start_migration('github', ['github/dotfiles'])
+      # @see https://developer.github.com/v3/orgs/migrations/#start-a-migration
+      def start_migration(org, repositories, options = {})
```

Comment:
Ugh, such shame. Looks like this was refactored out of existence: https://github.com/octokit/octokit.rb/commit/14542b81e5108c8d0fc85e8b419dc06dc378bf72

Sending a PR now.